### ISSUE TITLE
TimeOut / stuck in loop (?) - 1.24.1 PDF-hul for attached file

### ISSUE THREAD

**Issue by asciim0** [openpreserve/jhove#646]
Using both UI and CLI (on Windows environment, tested on different machines) the attached file nevers seems to reach a result for validation.

[897714407.pdf](https://github.com/openpreserve/jhove/files/5202088/897714407.pdf)

**Comment by rosetta-development** [comment]
We encounter the same issue with our pdf file.
Thread is stuck on : 
`edu.harvard.hul.ois.jhove.Property.getByName(Property.java:159)`

**Comment by karenhanson** [comment]
I think I have this same issue  - so wanted to share what I found while trying to troubleshoot in case it adds useful context. Looks like #645 may be the same thing too. In my case the following lines in the PDF are causing an infinite loop (using v1.24):
```
464 0 obj
<</Dest(þÿ s e c S 0 0 4)/Next 463 0 R/Parent 399 0 R/Prev 465 0 R/Title(þÿ P a r t A\t & \t .)>>
endobj
465 0 obj
<</Dest(þÿ s e c S 0 0 3)/Next 464 0 R/Parent 399 0 R/Prev 466 0 R/Title(þÿ P a r t B)>>
endobj
```
In the `Tokenizer` it seems that the backslash in the title causes it to go into `Literal.readBackslashSequence()` where it fails to see the `>>` that should end the entry. It then proceeds to read in the following row as part of the same object, which sets the `Next` value back to 464 and causes an infinite loop where it keeps reloading the same garbled object. In my case the PDF gets stuck [in this loop](https://github.com/openpreserve/jhove/blob/47f077fcb09d3dd5720aeeb4abf27de5dac48d19/jhove-modules/pdf-hul/src/main/java/edu/harvard/hul/ois/jhove/module/pdf/AProfile.java#L835-L841).

I note that the PDF attached to this issue and the one attached to #645 both have backslashes in the Title property of a dictionary entry. I tested the fix in PR #652 and it works for my issue too.
